src/shared/common/components/side-bar.jsx

```diff
@@ -129,7 +174,8 @@ class SideBar extends Component {
                             })
                         }
                     </ul>
-                    {this.renderActiveTab()}
+                    {this.state.renderAll && this.renderAllTabs()}
+                    {!this.state.renderAll && this.renderActiveTab()}
```

Comment:
I think it would be clearer that this is an either/or choice with a ternary operator
```suggestion
                    {this.state.renderAll ? this.renderAllTabs() : this.renderActiveTab()}
```